main/saturn-node.js

```diff
@@ -70,8 +70,8 @@ function getSaturnBinaryPath () {
 }
 
 async function start (/** @type {Context} */ ctx) {
-  if (!getFilAddress()) {
-    console.info('Saturn node requires FIL address. Please configure it in the Station UI.')
+  if (!wallet.getAddress()) {
+    console.info('Saturn node requires FIL address')
```

Comment:
This should never happen, right?

I am proposing to change `console.info` to `console.error`, or even better use electron-log with error severity to capture this information in the log file for future inspection.

It makes me wonder if we should throw an exception or exit the station completely if we ever get here.

Reply:
Based on the timing of calls in `main/index.js`, this can never happen indeed. The condition is here to satisfy the type system.

I think you're right that Station should crash hard here. I've changed it to a `throw`.